app/js/form.js

```diff
@@ -0,0 +1,183 @@
+
+$('input,textarea').on('input',function(){
+  var inpObj1 = document.getElementById("name").checkValidity();
+  var inpObj2 = document.getElementById("email").checkValidity();
+  var inpObj3 = document.getElementById("age").checkValidity();
+  var inpObj4 = document.getElementById("dropdown").checkValidity();
+  var inpObj5 = document.getElementById("most-like").checkValidity();
+  var inpObj7 = document.getElementById("comment").checkValidity();
+  console.log(inpObj7);
+  var requiredCheckboxes = $('input[type="checkbox"]');
```

Comment:
Make sure you remove console before publishing your work

Reply:
Done